orchestrator/gravity_utils/src/types/valsets.rs

```diff
@@ -339,7 +341,13 @@ impl Ord for ValsetMember {
         if self.power != other.power {
             self.power.cmp(&other.power)
         } else {
-            self.eth_address.cmp(&other.eth_address).reverse()
+            // Since gravity-bridge and ethers use checksum addresses
+            // we need to convert in order to do the comparison.
+            let checksum_addr = checksum::checksum(
+                &self.eth_address.unwrap_or_default().to_hex());
+            let checksum_other = checksum::checksum(
+                &other.eth_address.unwrap_or_default().to_hex());
+            checksum_addr.cmp(&checksum_other).reverse()
```

Comment:
and ethers doesn't implement comparison on addresses?

Reply:
you mean an hex address (hash) or checksummed address?

Reply:
checksummed addressed -- the comments says ethers uses checksummed addresses, so why to export to hex and compute a checksum address if the original was a checksummed address?
or is the comment incorrect that gravity bridge uses checksummed addresses, but ethers doesn't?

Reply:
I removed the comment on ethers which may be misleading